lib/grpnice.gi

```diff
@@ -836,6 +836,13 @@ end);
 AttributeMethodByNiceMonomorphism( Size,
     [ IsGroup ] );
 
+#############################################################################
+##
+#M  StructureDescription( <G> )
+##
+AttributeMethodByNiceMonomorphism( StructureDescription,
```

Comment:
Indeed, this duplicates the `StructureDescription` method already in `grpnames.gi`. Though I'd argue that the latter should be replaced by this. Using `AttributeMethodByNiceMonomorphism` is arguably the right thing to do.